modules/oe_content_entity/modules/oe_content_entity_organisation/modules/oe_content_organisation_reference/oe_content_organisation_reference.info.yml

```diff
@@ -0,0 +1,19 @@
+name: OpenEuropa Organisation Reference
+description: Allows to reference organisations in very specific cases, such as a contact information.
+package: OpenEuropa
+type: module
+core: 8.x
+
+dependencies:
+  - oe_content:oe_content
+  - oe_content:oe_content_entity
```

Comment:
Can be removed as oe_content:oe_content_entity_contact and oe_content:oe_content_organisation already had this deps.